WFP-VAM/prism-app
# Fix timelineitems display month translations COUNTRY=mongolia

This fixes issue #690 

How to test the feature:
- Select a layer on PRISM
- check the calendar month in english.
- Switch to MN language.

Screenshot/video of feature:

![NqnH6Lng1d](https://user-images.githubusercontent.com/3285923/212840928-34161a6d-6d40-48d7-92f2-e46f4112d301.gif)
